lib/sql-string.js

```diff
@@ -193,14 +193,16 @@ SqlString.bufferToString = function(buffer, dialect) {
 };
 
 SqlString.objectToValues = function(object, timeZone) {
-  var values = [];
-  for (var key in object) {
```

Comment:
I think good readable code is always important. As some of my changes here are just about improving readablitiy ;). But I also think it's important to have fast code. Especially in case of an ORM.
The solution using `while pop` should be quite nice to read, don't you think so? That's one line of extra code for the keys but you still keep the variable name as in the for in loop.